credentials/tls/certprovider/store_test.go

```diff
@@ -22,26 +24,39 @@ import (
 	"context"
 	"crypto/tls"
 	"crypto/x509"
+	"errors"
 	"fmt"
 	"io/ioutil"
 	"reflect"
 	"testing"
 	"time"
 
 	"google.golang.org/grpc/internal/grpctest"
+	"google.golang.org/grpc/internal/testutils"
 	"google.golang.org/grpc/testdata"
 )
 
 const (
 	fakeProvider1Name  = "fake-certificate-provider-1"
 	fakeProvider2Name  = "fake-certificate-provider-2"
 	fakeConfig         = "my fake config"
+	providerChanSize   = 2
```

Comment:
Why 2?

Reply:
In a couple of tests, we have a single builder returning two providers (because either the configs are different or because the certName is different).

Reply:
Couldn't those tests do a Receive() after each call to `GetProvider`?